File: ete3/read_newick.py
#!/usr/bin/env python

from ete3 import Tree
import os
import sys
import time

# Get input
treefile=sys.argv[1]
print "reading",treefile

# Start the clock
start = time.time()

# Run run run
t = Tree( treefile )

# Stop the clock
end = time.time()
print("Internal time: ", end - start)

File: ggtree/read_newick.R
#!/usr/bin/env Rscript

#library(tidyverse)
library(ggtree)

# Get input tree file
args <- commandArgs(trailingOnly = TRUE)
treefile <- args[1]
print(paste("reading ",treefile))

# Start the clock
start <- Sys.time()

# Run, Forrest, Run!
tree <- read.tree(treefile)

# Stop the clock
end <- Sys.time()
duration <- difftime(end, start, units="secs")
print(paste("Internal time:", duration))
